src/forms/_checkbox.scss

```diff
@@ -0,0 +1,33 @@
+.checkbox {
+  display: none;
+
+  + .checkbox__label::before {
+    content: '';
+    position: absolute;
```

Comment:
We have a `@include before()` mixin that will cover the `content: ''` and `position: absolute` values:
https://github.com/CasperSleep/nightshade-core/blob/master/src/phantom/utilities/_visual_utilities.scss#L9-L18

Example of usage: https://github.com/CasperSleep/Casper/blob/master/app/assets/stylesheets/modules/products/_specs.scss#L18-L22